README.md

```diff
@@ -523,3 +524,21 @@ python downloadIsisData --config rclone.conf <mission> $ISISDATA
 ```
 
 > The script does not support python2, sometimes you need to explicitly use python3 with `python3 downloadIsisData <mission> $ISISDATA --config rclone.conf` 
+
+## Semantic Versioning and It's Role in Describing the Software
+In 2019 the ISIS project adopted [semantic versioning](https://semver.org/) via it's second [Request for Comment](https://github.com/USGS-Astrogeology/ISIS3/wiki/RFC2:-Release-Process) (RFC). Semantic versioning was adopted as a tool to help quickly describe how changes to the software impact users and developers. Versions of ISIS are now descried using a Major.Minor.Bug scheme (e.g., 7.1.0). 
+
+For users and developers the first (Major) and second (Minor) version numbers are the most important. The final (Bug) number is incremented whenever one or more bug fixes are included in a version. Neither users nor developers should see any changes in the way ISIS programs are called or how the [API](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/APIdefinition.md) operates as the final (Bug) number increments.
+
+The first two numbers indicate whether the change(s) are breaking or non-breaking. What is a [breaking change](https://github.com/USGS-Astrogeology/ISIS3/wiki/RFC2:-Release-Process#terms)? If a change to [the API](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/APIdefinition.md), defined as programs (e.g., `spiceinit` or `cam2map` or `pds2isis`) and some text output (e.g., CSV output, but not `.txt`), alters the way in which a user calls the program or parses the program output in a way that a scripted solution would fail, that change in breaking. In other words, if a CSV output file removed or renamed a column, that would be breaking. If a CSV file added a new column, that would not be breaking. Likewise, if an application `spiceinit` adds a new arguments, that is non-breaking. If the change removes, reorders, or changes the way the application (CLI) is called, the change is breaking.
```

Comment:
`...fail, that change in breaking.` suggesting something like `...fail, that would be considered a breaking change.`